renovate.json5

```
@@ -48,6 +48,15 @@
         "k8s.io/component-helpers"
       ]
     },
+    {
+      // Group diskrsync and spgz dependent library Docker and go updates
+      // together in a single PR
+      "groupName": "diskrsync version",
+      "groupSlug": "diskrsync",
+      "matchPackagePrefixes": [
+        "github.com/dop251/"
+      ],
+    },
```

```suggestion
    {
      // Group diskrsync and spgz dependent library Docker and go updates
      // together in a single PR
      "groupName": "diskrsync packages",
      "groupSlug": "diskrsync",
      "matchPackagePatterns": [
        // Using the "packagePatterns" option so that we can match both the go
        // packages (w/o the "github.com/" prefix) and the other references that
        // do have it.
        "dop251/diskrsync",
        "dop251/spgz"
      ]
    },
```

The above is what I had to do to group all the instances of diskrsync and spgz together